src/lib/components/VideoEmbed.svelte

```diff
@@ -1,9 +1,21 @@
 <script lang="ts">
 	import FrameEmbed from '$lib/components/FrameEmbed.svelte'
 	import Splash from '$lib/components/Splash.svelte'
+	import { VideoSource } from '$lib/data'
 	import type { Video } from '$lib/data'
+	import { preferredSource } from '$lib/store.js'
 
 	export let video: Video
+
+	let videoSource: VideoSource
+
+	preferredSource.subscribe((value: VideoSource) => {
```

Comment:
This `let` and subscription can be removed in favor of using the store accessor syntax in the html, like so: `$videoSource`

Reply:
That was a lot more elegant, thank you.